matplotlibrc.template

```diff
@@ -359,6 +359,7 @@
                               ##     left, right and center
 #axes.titlesize     : large   ## fontsize of the axes title
 #axes.titleweight   : normal  ## font weight of title
+#axes.titlecolor    : auto    ## color of the axes title
```

Comment:
I think if we make `auto` the default, the comment should say what it inherits...